### ISSUE TITLE
Original docker build had clear-execstack and stripping - new one does not have

### ISSUE THREAD

**Issue by matiwinnetou** [eclipsesource/J2V8#311]
We are getting warnings that J2V8 has been complied without execstack -c. 

from original docker docker/Dockerfile.linux:
```
    execstack -c libj2v8_linux_x86_64.so && \
    echo "striping symbols...." && \
    strip --strip-unneeded -R .note -R .comment libj2v8_linux_x86_64.so && \
```

@drywolf shall we include this in new build system?

**Comment by drywolf** [comment]
> shall we include this in new build system?

Yeah, I have a point about it on my todo list. Thanks for mentioning.
I'll give it a higher priority.

**Comment by matiwinnetou** [comment]
@drywolf thanks I may also contribute but now I really want to fix #306 as well, following fixes for #310 and #308.

Docker builds:
Feedback from local folks was that is is probably better to use Ubuntu LTS for building docker images, perhaps there is a way to install cmake 3.6 from ubuntu-backports.

Our warning in PROD:
```
Java HotSpot(TM) 64-Bit Server VM warning: You have loaded library /mnt/mesos/sandbox/pubse/libj2v8_linux_x86_64.so which might have disabled stack guard. The VM will try to fix the stack guard now.
It's highly recommended that you fix the library with 'execstack -c <libfile>', or link it with '-z noexecstack'.
```

**Comment by drywolf** [comment]
@matiwinnetou Thanks for the hints.
I am currently working on this ... for the part about using Ubuntu LTS ... I am all for it, according to the ubuntu wiki the current LTS release is [Xenial](https://wiki.ubuntu.com/Releases), so that's what I would put into the docker files now. CMake would be installed manually again, but I don't see a problem there ... that's how it was in the first place anyway.

**Comment by matiwinnetou** [comment]
@drywolf Yeah indeed, I have to confess I changed my mind mostly due to issues with linking GCLIB library. We had exception in our integration system where we run debian jessie due to linking issues. Previous build was based on Debian Jessie.

Ubuntu 16.04 LTS is already a stretch goal but our new setup with new infrastructure supports docker containers so in theory we can pick newer distrubutions for individual applications. 

CMake will have to be sadly installed manually or we find some backport.

Do you know if others software like maven we can piggy back on distro's software rather than installing own versions?

**Comment by matiwinnetou** [comment]
"Stripped and non-stripped symbols for some / all platforms (this reduces the size, but makes debugging harder), I think this should be a configuration param to python script." We probably would not want to strip symbols now because library is not 100% stable yet and it really help to have more debugging info now.
